liveclasses/views.py

```
@@ -0,0 +1,53 @@
+import uuid
+from django.contrib.auth.mixins import LoginRequiredMixin
+from django.urls import reverse
+from django.views.generic import ListView, CreateView, View
+
+
+class LiveClassListView(LoginRequiredMixin, ListView):
+    model = LiveClass
+    template_name = "liveclasses/list_live_classes.html"
+    context_object_name = "live_classes"
+
+    def get_queryset(self):
```

Ruff (0.8.2)

Missing return type annotation for public function `get_queryset`

(ANN201)